package.json

```diff
@@ -39,16 +40,16 @@
     "@commitlint/cli": "^18.6.0",
     "@commitlint/config-conventional": "^18.6.0",
     "@commitlint/cz-commitlint": "^18.6.0",
-    "@storybook/addon-a11y": "^7.4.1",
-    "@storybook/addon-essentials": "^7.4.1",
-    "@storybook/addon-interactions": "^7.6.17",
-    "@storybook/addon-links": "^7.4.1",
+    "@storybook/addon-a11y": "7.6.20",
+    "@storybook/addon-essentials": "7.6.20",
+    "@storybook/addon-interactions": "7.6.20",
+    "@storybook/addon-links": "7.6.20",
```

Comment:
Can you update these to be `^7.6.20` like the old ones were `^7.4.1`? We try not to pin to exact versions unless we have to.